cmd/mirror-main.go

```diff
@@ -135,6 +142,9 @@ EXAMPLES:
 
   10. Mirror a bucket older than 30 days from Amazon S3 bucket test to a local folder.
       $ {{.HelpName}} --older-than=30 s3/test ~/test
+	
+  11. Mirror server encrypted objects from Minio cloud storage to a bucket on Amazon S3 cloud storage 
+      $ {{.HelpName}} --encrypt-key "minio/photos=32byteslongsecretkeymustbegiven1 , s3/archive=32byteslongsecretkeymustbegiven2" minio/photos/ s3/archive
```

Comment:
is the space needed before `,` and after? we can simplify the examples without space to avoid copy-paste mistakes.

Reply:
done.